reports/final/0-Title/4-List-of-Figures.md

```diff
@@ -16,6 +16,7 @@
 | [10. System Deployment](#fig-system-deployment)    | Jeremy               |                      |
 | [11. Layer Architecture](#fig-layer-architecture)  | Cameron              |                      |
 | [12. Client-Server Deployment](#fig-client-server) | Cameron              |                      |
+| [13. demo-layout](#fig-demo-layout)                | Peter                |                      |
```

Comment:
The first square bracket value is the name of the figure. This should be title case and should be reflective of the content in the figure. `Demo Layout` would be appropriate.